tests/jvm/src/test/resources/metac-metacp.diff

```diff
@@ -494,6 +494,7 @@ example/Synthetic#x$2.
  access {
 -  public_access {
```

Comment:
What caused this diff?

Reply:
Thats not a diff caused by this PR, its just github rendering it with a red outline :sweat_smile:

Reply:
lololol

![Screenshot 2021-03-03 at 16 40 14](https://user-images.githubusercontent.com/1408093/109834716-c9303e80-7c42-11eb-9d9f-aa3f59781dce.png)


This diff file always trips me up during review.